fix(serverless functions): update reviewed doc MTA-3234 (#1704)
scaleway/docs-content: serverless/functions/quickstart.mdx

  - serverless
---

> The Scaleway [Elements Functions platform](https://www.scaleway.com/en/serverless-functions/) makes your functions available, executes them on demand and manages resource allocation for you. A function is a lightweight unit of logic, defining a procedure for changing one element into another. The function remains static, while the variables that pass through it can vary. We show you how to create a functions namespace, create your function, and delete it when you no longer need it.

Reviewer: ```suggestion
The Scaleway [Serverless Functions platform](https://www.scaleway.com/en/serverless-functions/) makes your functions available, executes them on demand, and manages resource allocation for you. A function is a lightweight unit of logic, that defines a procedure for changing one element into another. The function remains static, while the variables that pass through it can vary. We show you how to create a functions namespace, create your function, and delete it when you no longer need it.
```

Tu peux consulter le Wording Handbook pour le naming officiel des produits: https://docs.google.com/document/d/1RvTZ0Wv2H4ZJ_CBmXAzmUICc_DWGwF_WyNR8eMF3S7o/edit

+ je pense qu'on utilise plus le mot "Elements", dans le doute on ne mets pas ;) hehe

<Message type="tip">

>   This page explains how to deploy functions using Scaleway Console but for advanced usage (ex: larger projects with dependencies, automation, multiple functions...) [several ways to deploy functions](/serverless/functions/reference-content/deploy-function/) are available.

Reviewer: ```suggestion
  This page explains how to deploy functions using the Scaleway Console. For advanced usage (ex: larger projects with dependencies, automation, multiple functions...) [several ways to deploy functions](/serverless/functions/reference-content/deploy-function/) are available.
```

    - Choose the **resources** to be allocated to your function at runtime. These define the performance characteristics of your function.
    - Set your **scaling** preferences, or leave them at default values. The Scaleway platform autoscales the number of available instances of your function to match the incoming load, depending on the settings you define here.
    - Define any **environment variables** you want to inject into your function. For each environment variable, click **+Add** and enter the key/value pair.
    - Set secret environment variables (optional). **Secrets** are environment variables which are injected into your function and stored securely, but not displayed in the console after initial validation. Add a **key** and a **value**.
    - Set the desired **privacy policy** for your function. This defines whether a function can be executed anonymously (**public**) or only via an authentication mechanism provided by the [Scaleway API](https://www.scaleway.com/en/developers/api/serverless-functions/#authentication) (**private**).
    - Click **Advanced Options** if you wish to set a custom timeout for your function.
      <Message type="note">

>         [Triggers](/serverless/functions/concepts/#trigger) can no longer be added from this screen. Refer to [this page](/serverless/functions/how-to/add-trigger-to-a-function/) to add a trigger to your function.

Reviewer: ```suggestion
        [Triggers](/serverless/functions/concepts/#trigger) can no longer be added from this screen. For more information, refer to the [How to add a trigger to a function](/serverless/functions/how-to/add-trigger-to-a-function/) documentation.
```
For accessibility reasons we try to include the page title in the hyperlink as much as we can 🤓
